web_src/js/components/RepoActionView.vue

```diff
@@ -74,7 +74,11 @@
                 <SvgIcon name="octicon-gear" :size="18"/>
               </button>
               <div class="menu transition action-job-menu" :class="{visible: menuVisible}" v-if="menuVisible" v-cloak>
-                <a class="item" :href="run.link+'/jobs/'+jobIndex+'/logs'" target="_blank">
+                <a class="item" :href="run.link+'/jobs/'+jobIndex+'/logs'" target="_blank" v-if="currentJob.steps.length>0">
+                  <i class="icon"><SvgIcon name="octicon-download"/></i>
+                  {{ locale.downloadLogs }}
+                </a>
+                <a class="item disabled" href="" target="_blank" v-else>
```

Comment:
The code just duplicates with above.

I don't think it's right to just copy&paste code.

You are using Vue so you need to use `:class={"disabled": shouldBeDisabled}`